packages/account/src/Sections/Verification/ProofOfIdentity/proof-of-identity-submission-for-mt5.jsx

```diff
@@ -72,16 +71,9 @@ const POISubmissionForMT5 = ({
 
         const submit_data = {
             identity_verification_document_add: 1,
-            document_number,
-            document_type: document_type.id,
-            issuing_country: citizen_data.value,
+            ...formatIDVFormValues(values, citizen_data.value),
         };
 
-        if (submit_data.document_type === IDV_NOT_APPLICABLE_OPTION.id) {
-            handlePOIComplete();
-            return;
-        }
-
         WS.send(submit_data).then(response => {
```

Comment:
Why are you removing this?

Reply:
The option to select "I want to do this later" is available only during Real account creation. The above snippet used to compare if the user has opted out of IDV and then prevent the invocation of `identity_verification_document_add` API call. But now we need to make the call irrespective of what the user has chosen under IDV dropdown